src/DataProtection/docs/Test-AzDataProtectionBackupInstanceReadiness.md

```diff
@@ -0,0 +1,323 @@
+---
+external help file:
+Module Name: Az.DataProtection
+online version: https://docs.microsoft.com/powershell/module/az.dataprotection/test-azdataprotectionbackupinstancereadiness
+schema: 2.0.0
+---
+
+# Test-AzDataProtectionBackupInstanceReadiness
```

Comment:
For this command, I think we only need to keep variant ValidateExpanded

Reply:
removed extra param sets